Binary files belle-sip-1.3.0-orig/src/grammars/.belle_sdp.g.swp and belle-sip-1.3.0/src/grammars/.belle_sdp.g.swp differ
diff -urN belle-sip-1.3.0-orig/src/grammars/belle_sdp.g belle-sip-1.3.0/src/grammars/belle_sdp.g
--- belle-sip-1.3.0-orig/src/grammars/belle_sdp.g	2014-10-03 18:58:33.672605009 -0400
+++ belle-sip-1.3.0/src/grammars/belle_sdp.g	2014-10-03 18:59:29.414376789 -0400
@@ -44,7 +44,9 @@
 
 #pragma GCC diagnostic ignored "-Wparentheses"
 #pragma GCC diagnostic ignored "-Wunused"
-#pragma GCC diagnostic ignored "-Wtautological-compare" 
+#ifdef __clang__
+#pragma GCC diagnostic ignored "-Wtautological-compare"
+#endif
 #ifndef __clang__
 #pragma GCC diagnostic ignored "-Wunused-variable"
 #endif
